components/ImageBanner/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,430 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+import "../index.scss";
+
+<Meta
+  title="CSS/Actions/Image Banner"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Image Banner" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <div class="denhaag-image-banner">
+      <div class="denhaag-image-banner__card">
+        <img
+          class="denhaag-image-banner__image"
+          src="https://images.unsplash.com/photo-1612799675078-ac368b17e488?w=507&h=353"
+          alt="alt text here"
+        />
```

Comment:
Meer een vraag dan een comment, moeten we voor deze images niet een <picture> en <source> element gebruiken om het voor verschillende screensizes beter te maken? Of doen we dat gewoon met de srcset op de <img> tag?